src/javaRosa/util.js

```diff
@@ -273,5 +292,6 @@ define([
         insertOutputRef: insertOutputRef,
         looksLikeMarkdown: looksLikeMarkdown,
         parseXLSItext: parseXLSItext,
+        outputToXPathOrHashtag: outputToXPathOrHashtag,
```

Comment:
What do you think of exposing two separate functions? It looks like all uses of this have hard-coded arguments. I'm thinking
```js
outputToHashtag: _.partial(outputToXPathOrHashtag, "toHashtag"),
outputToXPath: _.partial(outputToXPathOrHashtag, "toXPath"),
```